### PR TITLE
Add example for github-contrib

### PR DESCRIPTION
**Pull request by RainbowMango**
Excellent tools @nikhita .

Thanks. 

I think if there is an example would be better. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,11 @@ github-contrib -token=<your-token> <org> <github-handle>
     	print version and exit
 ```
 
+Example: get contribution list for `nikhita` against `kubernetes` community.
```
Comment (nikhita) [inline README.md:50]: Can you add this as a sub-section under Usage?

File: README.md [hunk 2]
```diff
@@ -47,6 +47,11 @@ github-contrib -token=<your-token> <org> <github-handle>
...
+```
+# github-contrib -token=5af802c9dc538c87c593e6fd173fa251a92cbcci kubernetes nikhita
```
Comment (nikhita) [inline README.md]: What token value is this? Hope it's not an active token :grimacing: 

Would prefer to leave the token value as just `<token>` or something.

Comment (RainbowMango) [reply README.md]: Don't be panic, not an active token. :)

File: README.md (hunk 1 above)
Comment (RainbowMango) [reply README.md:50]: I'd prefer to separate `Example` with `Usage`.

Because the `Example` seems dump from code. 

Do you mean add Example to code?

Comment (RainbowMango) [reply README.md:50]: You'd like this? 
```
github-contrib : v0.1.0
USAGE:
github-contrib -token=<your-token> <org> <github-handle>
  -token string
    	Mandatory GitHub API token
  -v	print version and exit (shorthand)
  -version
    	print version and exit
Example: 
# github-contrib -token=<token> kubernetes nikhita
```

Comment (nikhita) [reply README.md:50]: Revisiting this, I think this is ok as-is. :)

File: README.md (hunk 2 above)
Comment (nikhita) [reply README.md]: Phew :)

Can we still replace this with just `<token>`? Even though it's not an active token, I'm concerned folks could try using it. :/

**Review by nikhita — COMMENTED** [review]
Sorry, this dropped from my radar! :bowing_woman: 

Just one more comment, but looks good to go otherwise. :+1:

File: README.md (hunk 2 above)
Comment (RainbowMango) [reply README.md]: Sure. :)

**Review by nikhita — APPROVED** [review]
Thanks for addressing that so quick! :smile:

**Pull request merged**